pkg/agent/secondarynetwork/init.go

```diff
@@ -74,38 +102,58 @@ func Initialize(
 	return nil
 }
 
-// TODO: check and update bridge configuration.
+func RestoreOVSBridge(secNetConfig *agentconfig.SecondaryNetworkConfig) {
+	if len(secNetConfig.OVSBridges[0].PhysicalInterfaces) == 1 {
+		iface, ifaceIPs, ifaceRoutes, err := util.GetInterfaceConfig(secNetConfig.OVSBridges[0].PhysicalInterfaces[0])
```

Comment:
We should get the OVS internal port config here?

Reply:
`PrepareHostInterfaceConnection` renames the phy interface, `eth0` -> `eth0~` and we add `eth0` as internal port to the bridge (new interface), so technically we are getting internal port config here.

I had to add the rename step in `PrepareHostInterfaceConnection` to make it usable at `ConnectUplinkToOVSBridge`.